### PR TITLE
Csv export and duplicates links

### PR DESCRIPTION
**Pull request by somexpert**
Adds links to the csv export and duplicate finding admin views. This will bump the version to 1.1.0

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review

File: accession/admin_views.py [hunk 1]
```diff
@@ -82,8 +82,15 @@ def print_view(request, app_label, model_name, object_id):
...
     try:
         model = models.get_model(app, model)
-        response = render_to_csv_response(model.objects.all())
     except (AttributeError, LookupError) as e:
         raise Http404(str(e))
 
-    return response
+    if request.GET:
+        kwargs = request.GET.dict()
+        objects_list = model.objects.filter(**kwargs)
```
Comment (ldko) [inline accession/admin_views.py:78]: If the user does a search before clicking to export a csv, the search query parameter will be in the kwargs and throw an error when you try to use it as a parameter to limit which records get exported. For example:
- Go to the label page at tfc/admin/accession/label/export_csv/.
- Do a search there to result in a URL like: tfc/admin/accession/label/?q=Wang
- Click Export to CSV then results in a URL like: tfc/admin/accession/label/export_csv/?q=Wang which throws error like:
  `FieldError at /tfc/admin/accession/label/export_csv/
  Cannot resolve keyword u'q' into field. Choices are: id, label, object``

**Comment by ldko** [comment]
Just noticed that when we moved to using [find_duplicates](https://github.com/unt-libraries/django-accession/blob/csv-export-and-duplicates-links/accession/utils.py#L11) we may have lost case insensitivity in our finding of duplicates.

**Comment by somexpert** [comment]
If we were to make it that way, then when searching for duplicates in the Donor model, for instance, donors who have common substrings would show up as duplicates. For instance, a donor with the last name Ross and a donor with the last name Gross would trigger a duplicate on Ross. Do we want that? Also, per @vphill, the link for finding duplicates isn't necessary, so it may be that we no longer need that functionality. Something to discuss.

File: accession/admin_views.py [hunk 2]
```diff
@@ -82,8 +82,15 @@ def print_view(request, app_label, model_name, object_id):
...
+    else:
+        objects_list = model.objects.all()
+
+    render_to_csv_response(objects_list)
```
Comment (damonkelley) [inline accession/admin_views.py]: We can get rid of this.

Comment (somexpert) [inline accession/admin_views.py]: ah, right, good catch.

File: README.md [hunk 3]
```diff
@@ -36,15 +36,31 @@ Installation
...
+    ```
+
+4. Include the URLs.
     ```python
         urlpatterns = [
             url(r'^admin/', include(admin.site.urls)),
             url(r'^tfc/', include('accession.urls')),
         ]
     ```
+    You may also add the csv export, duplicates, and print urls from the app to
```
Comment (damonkelley) [inline README.md]: Do we still have a `duplicates` url?

Comment (somexpert) [inline README.md]: ah, left a straggler behind. nope, there should be no mention of duplicates anywhere. I'll get rid of it in this comment

**Comment by damonkelley** [comment]
:+1:

**Comment by ldko** [comment]
:+1:

**Pull request merged**